deposit/models.py

```diff
@@ -134,6 +157,25 @@ def __str__(self):
     class Meta:
         verbose_name_plural = 'Repositories'
 
+class LicenseChooser(models.Model):
+    """
+    Intermediate model to connect License to Repository Model
+    """
+    #: FK to license
+    license = models.ForeignKey(License, on_delete=models.CASCADE)
+    #: FK to repository
+    repository = models.ForeignKey(Repository, on_delete=models.CASCADE)
+    #: Identifier used for transmission
+    transmit_id = models.CharField(max_length=255, blank=False)
+    #: True if default license for this repository
+    default = models.BooleanField(default=False)
+
```

Comment:
I wonder if we should have a field, either here or in the `License` model, to make it possible to sort licenses.
We probably want to be able to control in which order the licenses are presented to the user (for instance, creative commons licenses should probably be grouped together).

I guess we can also decide to use the primary key of this model for that. We can define the order for each repository by creating the `LicenseChooser` objects in the desired order (it should not be a problem to delete them all and recreate them, as no FK points to them, right?)

Reply:
Currently the order is hard-coded in alphabetical order in the `Protocol` class, so for now all the CCs are grouped together and it's relative stable. But having the order customized with a simple system, i.e. entering any number for each LicenseChooser object is a good idea. A possible "No license" or "Other license" as last option seems reasonable to me.

Using PK is no good. Adding license "A" means to delete and recreate alls LicenseChooser for all repos that use license A, assuming you want alphabetical order.